src/app/workflow/workflow.component.html

```diff
@@ -96,6 +96,15 @@ <h3>
           >
             View Public
           </a>
+          <button 
+            mat-flat-button
+            id="DOIButton"
+            class="mr-2"
+            color="accent"
+           (click)="requestDOIForWorkflowVersion()"
+           [disabled]="(isRefreshing$ | async) || selectedVersion?.doiURL"
```

Comment:
Should this be disabled if the user has not linked their Zenodo account?

Alternatively, we had talked about this redirecting the user to link their Zenodo account.

I suggest disabling for now if the account is not linked, with a tooltip explaining why. Then another ticket for the preferred solution which would be to redirect to link.

Reply:
Need pointer on how to disable...

Reply:
I replied to you on Slack, but you are presumably already doing something similar on the accounts page, which you display Link/Unlink account button based on whether they have Zenodo creds or not.

Reply:
Next sprint https://ucsc-cgl.atlassian.net/browse/SEAB-342